buildSrc/src/main/kotlin/config/AndroidLibraryConfig.kt

```diff
@@ -24,15 +25,15 @@ fun AndroidLibraryExtension.setDefaults(generateBuildConfig: Boolean = false) {
         this.buildConfig = generateBuildConfig
     }
     sourceSets.getByName("main") {
-        manifest.srcFile("src/androidMain/AndroidManifest.xml")
+        if (project.file("src/androidMain/AndroidManifest.xml").exists()) {
+            manifest.srcFile("src/androidMain/AndroidManifest.xml")
+        }
```

Comment:
```suggestion
        val srcDir = manifest.srcFile.parentFile.parentFile
        srcDir.resolve("androidMain/AndroidManifest.xml").let {
            if (it.exists()) manifest.srcFile(it)
        }
```
This alternative doesn't require to add the `project` parameter (which can then be removed as unused by the IDE).